nri-config-generator/main.go

```diff
@@ -76,13 +76,10 @@ func main() {
 		os.Exit(0)
 	}
 	httport.SetPrometheusExporterPort("localhost", port)
-	// long running execution
+	// long running execution. This is a long running execution because in case of the export port was auto-generated (a random port incase
+	// whe the exporter_port is not provided by configuration) we need to keep this port for the exporter.
```

Comment:
```suggestion
	// long running execution. This is a long running execution because in case of the export port was auto-generated (a random port in case
	// the exporter_port is not provided by configuration) we need to keep this port for the exporter.
```